src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/ListCaseNotificationHandler.java

```diff
@@ -86,25 +87,28 @@ public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCa
         final String caseId = asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class)
             .orElseThrow(() -> new IllegalStateException("Case ID for the appeal is not present"));
 
+        boolean adjournHearingWithoutDate = (callback.getEvent()
+            .equals(Event.ADJOURN_HEARING_WITHOUT_DATE));
+
         final HearingInstructMessage bundleInstructMessage =
             hearingInstructMessage()
                 .withConsumerReference(notificationsHelper.getConsumerReference(caseId))
                 .withHoReference(homeOfficeReferenceNumber)
                 .withMessageHeader(notificationsHelper.getMessageHeader())
                 .withMessageType(HEARING.name())
-                .withNote(getHearingNotificationContent(asylumCase))
-                .withHearing(getHearingData(asylumCase))
+                .withNote(getHearingNotificationContent(asylumCase, adjournHearingWithoutDate))
+                .withHearing(getHearingData(asylumCase, adjournHearingWithoutDate))
                 .build();
 
         log.info("Finished constructing {} notification request for caseId: {}, HomeOffice reference: {}",
```

Comment:
These logging would be misleading for edit listing and adjourn. This should go away when you refactor as per the above comment

Reply:
Created own handler and refactored to use common code. Had to extend as sonar was complaining on code duplication.